### DISCUSSION TITLE
Database Connection Error: 3D000 undefined:undefined

### DISCUSSION THREAD

**Discussion by Bambiee** [requarks/wiki#3967] [Help / Questions]
What is this thing that prevents me from going forward??

Database Connection Error: 3D000 undefined:undefined

I'm trying to install it on mac, and it sent me this error at the final step: node server

**Comment by ghost** [comment]
Can you show us a template of your configuration file?

**Comment by Bambiee** [comment]
i can send you a screenshot of it... im not a tech guy...


> Can you show us a template of your configuration file?

i can send you a screenshot of it... im not a tech guy...
link:https://drive.google.com/file/d/1CCmoSWwzFYqWMtLnVHqZBO63jMXzJN8S/view?usp=sharing

**Comment by xstable** [comment]
Have the same issue, did you solve it at least?

**Comment by tigerinus** [comment]
Got the same issue... anyone please help?

**Comment by markuskreth** [comment]
check the database configuration settings.

**Comment by HemenRohani** [comment]
To solve this problem create an empty database.
